src/uwtools/templater.py

```diff
@@ -83,22 +83,22 @@ def parse_args(argv):
 
 def set_template(argv):
     '''Main section for rendering and writing a template file'''
-    logfile = os.path.join(os.path.dirname(__file__), "templater.log")
-    log = Logger(level='info', logfile_path=logfile)
-
-
     user_args = parse_args(argv)
 
+    logfile = os.path.join(os.path.dirname(__file__), ".logtemplater")
```

Comment:
I think `templater.log` was a bit more standard. Any specific reason for the change?

Reply:
To be honest, I did see a good reason to change it, and now I can't for the life of me remember where or why,  so it can't be that good of a reason! I'll revert this